### ISSUE TITLE
CSP Error after opening some pages immediately following update to 1.15.0-beta.1

### ISSUE THREAD

**Issue by WizWareTech** [humhub/humhub#6484]
### What steps will reproduce the problem?
when first opening a page, everything is fine. after a refresh (like F5), the problem occurs. If I then do a CTRL-F5 the problem goes away. Standard F5 causes problem. It is very easy to reproduce and never did this prior to this update.
### What is the expected result?
I expect all scripts to work properly in the code and not to see CSP script errors.
### What do you get instead?
The tab changes its name to 'Undefined' then anything that requires scripts stops working and this shows in the log:

Content Security Policy blocks inline execution of scripts and stylesheets
The Content Security Policy (CSP) prevents cross-site scripting attacks by blocking inline execution of scripts and style sheets.
To solve this, move all inline scripts (e.g. onclick=[JS code]) and styles into external files.
Allowing inline execution comes at the risk of script injection via injection of HTML script elements. If you absolutely must, you can allow inline script and styles by:
adding unsafe-inline as a source to the CSP header
adding the hash or nonce of the inline script to your CSP header.

Directive | Element | Source Location | Status
-- | -- | -- | --
script-src-elem |   | information:43 | blocked
script-src-elem |   | information:229 | blocked
script-src-elem |   | information:394 | blocked

### Additional info

| Q                | A
| ---------------- | ---
| HumHub version   | 1.15.0-beta.1
| PHP version      | 7.4.28
| Operating system | Linux

> Note: Also provide the full error stacks from `Administration --> Information --> Logging` and errors from your browser console if possible.
> Note: Please use the [community](https://community.humhub.com) site for any questions on customizing HumHub.

**Comment by WizWareTech** [comment]
I was able to dig down a little further and found that if I disabled nonce in web.php, the problem goes away. The strange thing is, it works fine on the initial page load (or CTRL-F5) and then not on subsequent load. So, I think there is some issue caused by the update related to the nonce. I don't know enough to know what would cause this, but I hope I've helped get someone close to the root cause.
